### DISCUSSION TITLE
Data Importer throws exception and does not overwrite the object :(

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#17869] [Q&A / Support]
![grafik](https://github.com/user-attachments/assets/83928620-82f1-4905-a028-b92ed87fce57)
The importer recognizes delta, but cant overwrite the object :/ "Duplicate Full Path"... yes. the name should be the same.... (system key)

**Comment by ghost** [comment]
What Information you need?

**Comment by kingjia90** [comment]
Thank you for reporting!

It looks like there are some issues in the configuration and is not set properly to handle the cases when an existing element is found.

Please have a look on https://github.com/pimcore/data-importer/blob/1.x/doc/03_Configuration/04_Resolver_Settings.md#element-loading
My guess it's currently set on [Loading Strategy: No Loading](https://github.com/pimcore/data-importer/blob/1.x/doc/03_Configuration/04_Resolver_Settings.md#loading-strategy-no-loading)

If it's a different issue, please share more details on how the configuration looks like, especially under the `Resolver` tab

**Reply by ghost** [reply]
Thank you very much :) It works

**Comment by ghost** [comment]
how i can clear the hashes the importer checks if delta check is activated? problem is that if i delete all imported data and reimport stuff it doesnt work because there is no delta

**Reply by kingjia90** [reply]
Thank you for your input, unfortunately i am not sure if i got it, i'd suggest to open a new discussion with more details, examples, screenshots, etc.. about the specific use case. 

IMHO Writing here as followup under an answered discussion thread, it tend to be less visible/engaged.
